jekyll/_cci2/configuration-reference.md

```diff
@@ -739,10 +739,70 @@ Key | Required | Type | Description
 version | Y | String | Should currently be `2`
 {: class="table table-striped"}
 
-#### **<`workflow_name`>**
+### **<`workflow_name`>**
 
 A unique name for your workflow.
 
+#### **`triggers`**
+Specifies which triggers will cause this workflow to be executed. Default behavior is to trigger the workflow when pushing to a branch. 
+
+Key | Required | Type | Description
+----|-----------|------|------------
+triggers | N | String | Should currently be `schedule`.
+{: class="table table-striped"}
+
+##### **`schedule`**
+A workflow may have a `schedule` indicating it runs at a certain time, for example a nightly build that runs every day at 12am UTC:
+
+```
+workflows:
+   version: 2
+   nightly:
+     triggers:
+       - schedule:
+           cron: "0 0 * * *"
+           filters:
+             branches:
+               only:
+                 - master
+                 - beta
+     jobs:
+       - test
+```
+###### **`cron`**
+The `cron` key is defined using POSIX `crontab` syntax.
+
+Key | Required | Type | Description
+----|-----------|------|------------
+cron | Y | String | See the [crontab man page](http://pubs.opengroup.org/onlinepubs/7908799/xcu/crontab.html).
+{: class="table table-striped"}
+
+###### **`filters`**
+Filters can have the key `branches`. 
+
+Key | Required | Type | Description
+----|-----------|------|------------
+filters | Y | Map | A map defining rules for execution on specific branches
+{: class="table table-striped"}
+
+####### **`branches`**
+
+The `branches` key controls whether the *current* branch should have a schedule trigger created for it, where *current* branch is the branch containing the `config.yml` file with the `trigger` stanza. That is, a push on the `master` branch will only schedule a workflow for the `master` branch.
+
+Branches can have the keys `only` and `ignore` which either map to a single string naming a branch (or a regexp to match against branches, which is required to be enclosed with /s) or map to a list of such strings.
+
+- Any branches that match `only` will run the job.
+- Any branches that match `ignore` will not run the job.
+- If neither `only` nor `ignore` are specified then all branches will run the job.
+- If both `only` and `ignore` are specified the `only` is considered before `ignore`.
+
+Key | Required | Type | Description
+----|-----------|------|------------
+branches | N | Map | A map defining rules for execution on specific branches
```

Comment:
Shouldn't `branches` be a required field?

Reply:
Yes, it should.

Reply:
ok, thanks, fixed